spid_cie_oidc/entity/models.py

```diff
@@ -263,7 +263,6 @@ def set_jwks_as_array(self):
                 setattr(self, i, [value])
 
     def save(self, *args, **kwargs):
-        self.entity_type = self.type[0]
```

Comment:
perché è rimosso? prende il primo entity type disponibile come quello default

Reply:
Perché quando si modifica l'EC di un RP nel db viene salvato il valore `federation_entity` a prescindere dal valore della select, che sta in `self.type[1]` e non viene preso